### ISSUE TITLE
`zip` artifacts with jlinked jre don't work on `arm64`

### ISSUE THREAD

**Issue by Christewart** [bitcoin-s/bitcoin-s#4383]
New issue as the docker portion of #4369 is fixed.

See this comment for the details: https://github.com/bitcoin-s/bitcoin-s/issues/4369#issuecomment-1149978653

**Comment by Christewart** [comment]
Related issue to get unblocked: https://github.com/actions/virtual-environments/issues/2552

**Comment by Christewart** [comment]
New issue: https://github.com/actions/runner-images/issues/5631

**Comment by Christewart** [comment]
I believe there is native support for this now: https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/
